### ISSUE TITLE
Confusing behavior around nested pseudo selectors

### ISSUE THREAD

**Issue by courier-new** [adrianlee44/atom-aligner-scss#3]
Hi! I love using aligner with my stylesheets but have been experiencing a bit of an annoyance around pseudo elements and classes. Take the following example:

```css
#container {
  display: block;
  background-color: red;
  a:hover {
    background-color: black;
  }
}
```

I am currently using the default settings for atom-aligner-scss. When the cursor is positioned somewhere around the attributes of `#container` and the "align" command is called, the result is the following:

```css
#container {
  display:          block;
  background-color: red;
  a
    background-color: black;
  }
}
```

The aligning works fine if the nested styles do not have a selector with ":" in them, of course. If I instead manually select just the attributes of `#container` and call "align" on them, I can avoid the issue. However, it slows down the aligning process significantly to need to keep checking if there is a nested selector with a ":" after the current styles or not. Wondering if there is any possibility to recognize and ignore such cases. Thanks!

**Comment by adrianlee44** [comment]
Thanks for reporting this. Taking a look at the issue.

**Comment by adrianlee44** [comment]
This should be fixed with updating aligner to v1.2.1.

**Comment by courier-new** [comment]
Great, thank you so much for addressing this so quickly! I will go and update right away! 👍
